use_case_examples/hybrid_model/compile_hybrid_llm.py

```diff
@@ -0,0 +1,110 @@
+"""Showcase for the hybrid model converter."""
+
+import json
+import os
+from copy import deepcopy
+from pathlib import Path
+from typing import List, Union
+
+import torch
+from concrete.fhe import Configuration, ParameterSelectionStrategy
+from datasets import load_dataset
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+from concrete.ml.torch.hybrid_model import HybridFHEModel
+
+
+def compile_model(
+    model_name: str,
+    model: torch.nn.Module,
+    inputs: torch.Tensor,
+    module_names: Union[str, List],
+    models_dir: Path,
+):
+    """Run the test for any model with its private module names."""
+    # Create a hybrid model
+    hybrid_model = HybridFHEModel(model, module_names)
+    # Compile hybrid model
+    hybrid_model.compile_model(
+        inputs,
+        n_bits=8,
+    )
+
+    # Save model for serving
+    models_dir.mkdir(exist_ok=True)
+    model_dir = models_dir / model_name
+    print(f"Saving to {model_dir}")
+    via_mlir = bool(int(os.environ.get("VIA_MLIR", 0)))
+    hybrid_model.save_and_clear_private_info(model_dir, via_mlir=via_mlir)
+
+
+if __name__ == "__main__":
+    configs = [
+        "transformer.h.0.mlp",  # Full MLP
+        ["transformer.h.0.mlp", "transformer.h.1.mlp"],  # Two full MLPs
+        "transformer.h.0.mlp.c_proj",  # only projection in MLP
+        "transformer.h.0.attn.c_proj",  # only projection in MLP
+    ]
+    config_index = int(os.environ.get("INDEX", 2))
+    config = configs[config_index]
```

Comment:
I find this a bit complicated to get the module names from a env variable in this case. I feel like compile.sh is not really useful and we could just launch compule_hybrid_llm.py directly with a arg.

Also may be worth to rename config at this point as it's actually module_names

Reply:
Anything that makes the user a bit more aware about what he launches.
Something like this?

```
python compile_hybrid_llm.py --private_module first_mlp/two_first_mlp/first_linear
```

Reply:
Moved to argparser

Reply:
thanks a lot